sharding/indexHashedNodesCoordinator.go

```diff
@@ -648,7 +648,11 @@ func (ihgs *indexHashedNodesCoordinator) computeNodesConfigFromList(
 		sort.Sort(validatorList(leavingList))
 	}
 
-	nbShards := core.MaxInt(0, len(eligibleMap)-1)
+	if len(eligibleMap) == 0 {
```

Comment:
I think that len(eligibleMap) should be greater than 1 as we should have at least 1 shard beside metachain, but you can leave it like this for now.